pages/_app.tsx

```diff
@@ -29,7 +61,38 @@ function MyApp({ Component, pageProps }: AppProps) {
               />
             </Head>
             <Navbar />
-            <Component {...pageProps} />
+            <Notification visible={!cookiesAccepted} severity="info">
```

Comment:
I think it would look cleaner as a separate component instead of something within _app.tsx

Reply:
We want this to be everywhere on the app if the guy didn't accept it

Reply:
I meant in the code, you could create a <PrivacyNotification /> component and import it

Reply:
Ok I see